### ISSUE TITLE
Debian release file is not valid yet

### ISSUE THREAD

**Issue by thinkh** [phovea/phovea_server#118]
* Release number or git hash: 7606043e14d9c9913956b4b9132ef3226c38719f
* Web browser version and OS: Linux
* Environment (local or deployed): local

### Steps to reproduce

1. `yo phovea:setup-workspace caleydo/taco_product -b develop`
2. `docker-compose up`

### Observed behavior

```
➜  taco docker-compose up
Creating network "taco_default" with the default driver
Building api
Step 1/11 : FROM python:3.7-buster
 ---> b567432174fe
Step 2/11 : LABEL maintainer="contact@caleydo.org"
 ---> Using cache
 ---> e2ceffd4d327
Step 3/11 : WORKDIR /phovea
 ---> Using cache
 ---> 59655c14cd27
Step 4/11 : COPY requirements*.txt docker_packages.txt docker_script*.sh _docker_data* ./
 ---> Using cache
 ---> 4e04c0f49723
Step 5/11 : RUN (!(test -s docker_packages.txt) || (apt-get -y update && cat docker_packages.txt | xargs apt-get install -y)) && 
    pip install --no-cache-dir -r requirements.txt &&     (pip install --no-cache-dir -r requirements_dev.txt)
 ---> Running in 3826ce3ab78f
Get:1 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:2 http://deb.debian.org/debian buster InRelease [122 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [49.3 kB]
Get:4 http://deb.debian.org/debian buster/main amd64 Packages [7907 kB]
Reading package lists...
E: Release file for http://security.debian.org/debian-security/dists/buster/updates/InRelease is not valid yet (invalid for another 12h 0min 14s). Updates for this repository will not be applied.
E: Release file for http://deb.debian.org/debian/dists/buster-updates/InRelease is not valid yet (invalid for another 1d 3h 49min 33s). Updates for this repository will not be applied.
ERROR: Service 'api' failed to build: The command '/bin/sh -c (!(test -s docker_packages.txt) || (apt-get -y update && cat docker_packages.txt | xargs apt-get install -y)) &&     pip install --no-cache-dir -r requirements.txt &&     (pip install --no-cache-dir -r requirements_dev.txt)' returned a non-zero code: 100
```

### Expected behavior

* No errors during build

**Comment by thinkh** [comment]
This error is related and caused by the last merged PR #116.

**Comment by thinkh** [comment]
[Restarting my local Docker Desktop](https://askubuntu.com/a/1167538) solved the problem!
